src/services/Bets/Bets.ts

```diff
@@ -4,8 +4,8 @@ import type { Bet } from '@/typings/BetTypes';
 import { BetStatusEnum } from '@/enum/BetEnum';
 
 class APIBets extends APIBase {
-  async getBets(): Promise<Bet[]> {
-    return this.get<Bet[]>('bets');
+  async getBets(limit = 10, page = 1): Promise<{ bets: Bet[], total: number, totalPages: number }> {
```

Comment:
this could be better, if you create the types including the paginations properties there, and also create a pagination type could be useful to be reused in the future